src/components/Chat/Chat.tsx

```diff
@@ -258,6 +267,22 @@ const ChatWithLLM: React.FC<ChatWithLLMProps> = ({
     fetchDefaultModel();
   };
 
+  const parseFlashcardAnswer = (
+    messageToBeParsed: string
+  ): { question: string; answer: string }[] => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [actualOutput, _fileReferences] = messageToBeParsed.split(
```

Comment:
this should be safe, even if the output doesn't contain file delimiter because it will by default return original output